qa-covid19.planx-pla.net/manifest.json

```diff
@@ -4,27 +4,27 @@
     "That's all I have to say"
   ],
   "versions": {
```

Comment:
the title says jcoin but this is covid19

Reply:
dam yes!

Reply:
🤦 ... sorry!